extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -349,13 +352,24 @@ export class UserNotificationManager {
   }
 
   /**
+   * Retrieves a registered user notification by ID. Returns undefined if it
+   * is not registered yet.
+   * @param {string} id
+   * @return {!Promise<?NotificationInterface>}
+   */
+  getNotification(id) {
+    return whenDocumentReady(this.win_.document).then(() => this.registry_[id]);
```

Comment:
Since this may be called multiple times, storing a `docReady` promise would be a little nicer.